plugins/ua_pubsub_ethernet.c

```diff
@@ -1049,7 +1057,8 @@ UA_PubSubChannelEthernet_send(UA_PubSubChannel *channel,
     /* copy payload of ethernet message */
     memcpy(ptrCur, buf->data, buf->length);
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,19,0)
+#if defined(KERNEL_VERSION)
```

Comment:
Merge the #if.

Reply:
I've done it on purpose in separate #ifs. At least my compiler (armclang 6.6) complains when I concatenate it. See

ua_pubsub_ethernet.c:830:55: error: function-like macro 'KERNEL_VERSION' is not defined
#if defined(KERNEL_VERSION) && (LINUX_VERSION_CODE >= KERNEL_VERSION(4,19,0))

Reply:
Alternatively we can define a wrapper macro that would do both checks in 1 line:

#if defined(KERNEL_VERSION)
#define LINUX_AND_KERNEL_VERSION_HIGHER_OR_EQUAL_4_19_0 (LINUX_VERSION_CODE >= KERNEL_VERSION(4,19,0))
#else
#define LINUX_AND_KERNEL_VERSION_HIGHER_OR_EQUAL_4_19_0 0
#endif

Reply:
Ok. Then lets keep as-is.